module-3/src/utils/SavedRecordRes.ts

```diff
@@ -0,0 +1,3 @@
+export interface SavedRecordRes {
```

Comment:
I don't think that util is a best place to store interfaces. You can create a separate directory to store all common types and interfaces